test/core_organization_test.go

```diff
@@ -70,23 +70,38 @@ func TestCoreOrganization(t *testing.T) {
 	require.NoError(t, cl.Create(ctx, org))
 	require.NoError(t, testutil.WaitUntilReady(ctx, cl, org))
 
-	projectTemplate := &corev1alpha1.OrganizationRoleTemplate{}
-	require.NoError(t, cl.Get(ctx, types.NamespacedName{
-		Name: templates.ProjectAdminOrganizationRoleTemplateName,
-	}, projectTemplate))
-
-	rbacTemplate := &corev1alpha1.OrganizationRoleTemplate{}
-	require.NoError(t, cl.Get(ctx, types.NamespacedName{
-		Name: templates.RBACAdminOrganizationRoleTemplateName,
-	}, rbacTemplate))
+	projectTemplate := &corev1alpha1.OrganizationRoleTemplate{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: templates.ProjectAdminOrganizationRoleTemplateName,
+		},
+	}
+	require.NoError(t, testutil.WaitUntilReady(ctx, cl, projectTemplate))
+	rbacTemplate := &corev1alpha1.OrganizationRoleTemplate{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: templates.RBACAdminOrganizationRoleTemplateName,
+		},
+	}
+	require.NoError(t, testutil.WaitUntilReady(ctx, cl, rbacTemplate))
+	require.NoError(t, cl.WaitUntil(ctx, org, func() (done bool, err error) {
+		if len(org.Status.Members) > 0 {
+			assert.Len(t, org.Status.Members, 1)
+			assert.Equal(t, org.Status.Members[0].Name, owner.Name)
+			return true, nil
+		}
+		return false, nil
+	}))
 
-	assert.Empty(t, org.Status.Members)
+	t.Log("Organization Owner has permission to create RoleBinding")
+	cfg.Impersonate = rest.ImpersonationConfig{
+		UserName: owner.Name,
+	}
+	ownerClient := testutil.NewRecordingClient(t, cfg, testScheme, testutil.CleanupOnSuccess)
```

Comment:
don't hardcode cleanup configuration, use the variable one.

Reply:
done